pkg/scheduler/queue/queue.go

```diff
@@ -348,7 +394,7 @@ func (q *RequestQueue) enqueueRequestInternal(r requestToEnqueue) error {
 //
 // The requestForQuerier message can contain either:
 // a) a query request which was successfully dequeued for the querier, or
-// b) an ErrShuttingDown indicating the querier has been placed in a graceful shutdown state.
```

Comment:
this was incorrect. The similarly named error for the RequestQueue service is ErrStopped and the one for the scheduler itself is ErrSchedulerIsNotRunning